sdks/docker-enclave/supervisord.conf

```diff
@@ -40,19 +40,23 @@ stderr_logfile_maxbytes=0
 
 # your custom programs go here
 
-# docker
-[program:docker]
+# Docker daemon
+[program:dockerd]
 command=/bin/dockerd --iptables=false
 autorestart=true
+startretries=3
```

Comment:
what is this for?

Reply:
was initially using this snippet for the build script which checked if docker-daemon has started and loaded images. will remove it for this.

Reply:
but I believe keeping it for the docker compose will be a good to have, just incase some sorta resource allocation fails. thoughts?
```
# Docker Compose
[program:compose]
command=docker compose -f /app/docker-compose.yml up --remove-orphans
autorestart=true
startretries=3
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stdout
stderr_logfile_maxbytes=0
depends_on=dockerd
```

Reply:
I think it's fine, let it keep trying, not like stopping has any point when the enclave is still occupied